ibm/service/kubernetes/resource_ibm_container_addons.go

```diff
@@ -58,10 +58,12 @@ func ResourceIBMContainerAddOns() *schema.Resource {
 				Description: "To manage all add-ons installed in the cluster using terraform by importing it into the state file",
 			},
 			"managed_addons": {
-				Type:        schema.TypeList,
-				Computed:    true,
-				Elem:        &schema.Schema{Type: schema.TypeString},
-				Description: "Used to keep track of the add-on names",
+				Type:             schema.TypeList,
```

Comment:
Why is computed variable change to optional?

Reply:
An error comes up when we try to update tf, so I made it optional

Reply:
I think that not the fix for this if you mark it optioan user can provide it as input